### ISSUE TITLE
Issue with HTML encoding

### ISSUE THREAD

**Issue by jnolis** [enricoschumann/textutils#1]
Hi!

I just downloaded this package in hopes of HTML encoding some text, but it seem that there is some issue causing the code to fail. This happened with both the CRAN version and when I updated to the version on your site.

```
> textutils::HTMLencode("hi")
Error in gsub(.html_entities[i + 1], .html_entities[i], x) : 
  invalid regular expression '\', reason 'Trailing backslash'
```

It seems like something to do with the .html_entities? Thanks!

**Comment by enricoschumann** [comment]
Thanks for the report. I have fixed the issue and pushed the change to GitHub. Would you give it a try, please?
```
> textutils::HTMLencode("hi")
[1] "hi"
> textutils::HTMLencode("hi &")
[1] "hi &amp;"
```

**Comment by jnolis** [comment]
It looks like it works now! Interestingly, when I reinstalled it using `devtools::install_github("enricoschumann/textutils")` the issue was still there, whereas when I used `install.packages('textutils', type = 'source', repos = c('http://enricoschumann.net/R', getOption('repos')))` it was fine. I'm not sure if that had something to do with me not clearing out the old version, but regardless my issue is resolved. Thanks for the quick turn around!
